src/renderer/entities/transaction/lib/validateBalance.ts

```diff
@@ -59,10 +59,11 @@ const validateBalanceForFee = async ({ transaction, getTransactionFee, api, ...p
   const transferableBalance = transferableAmount(tokenBalance);
   const transferableNativeTokenBalance = transferableAmount(nativeTokenBalance);
   const fee = await getTransactionFee(transaction, api);
+  const feeBN = new BN(fee).add(new BN(transaction.args.xcmFee || 0));
```

Comment:
`fee` is paid in origin chain's comission assets and the xcmFee is paid in sending asset. I dont think its right to add them to use in validations